src/include/access/csn_log.h

```diff
@@ -15,6 +15,18 @@
 #include "storage/sync.h"
 #include "utils/snapshot.h"
 
+#define InProgressXidCSN	 	UINT64CONST(0x0)
```

Comment:
Why? Is there a specific reason?

Seems like we are simply replacing "access/csn_snapshot.h" with ""access/csn_log.h" in transam.h 
If this is truly required, we should also change the same in neon part.

Reply:
The more I think about it; I think it makes sense to keep the CSN log and snapshot logic separate. Specifically, all these constants and functions are related to the "snapshot" of the region. On the other hand, the csn_log is only concerned with writing out the SLRU CSN mapping to the persistent log.

Reply:
True. I'll revert this change. Still, I think "csn snapshot" is not a great name for that module anymore because most of the functions there deal with assigning the CSN and XID->CSN mapping. But we can keep the name for now.